test/testData/mockReportData.json

```diff
@@ -251,7 +260,119 @@
+        }
+    ],
     "kbMatches": [
+        {
+            "approvedTherapy": false,
+            "category": "biological",
+            "context": "APC",
+            "kbStatementId": "#155:10801",
+            "kbVariant": "APC frameshift mutation",
+            "variant": "fe29d4e8e7b7f60751c880d71dd3595b",
+            "variantType": "mut",
+            "kbVariantId": "#161:769",
+            "matchedCancer": false,
+            "reference": "Inferred Functional Annotation",
+            "relevance": "likely loss of function",
+            "externalSource": "Inferred Functional Annotation",
+            "kbData": {}
+        },
+        {
+            "approvedTherapy": false,
+            "category": "therapeutic",
+            "context": "tankyrase inhibitor",
+            "disease": "colorectal cancer [DOID:9256]",
+            "kbStatementId": "#155:840",
+            "kbVariant": "APC:p.T1556fs",
+            "variant": "a12458ff7c8eb865656fd61f670cd0d5",
+            "variantType": "mut",
+            "kbVariantId": "#158:1343",
+            "matchedCancer": true,
+            "reference": "pmid:28179481",
+            "relevance": "resistance",
+            "externalSource": "IPRKB",
+            "externalStatementId": "2c463a78-efdc-4858-b378-250096ba086e",
+            "reviewStatus": "pending",
+            "kbData": {}
+        },
+        {
+            "approvedTherapy": false,
+            "category": "therapeutic",
+            "context": "test inhibitor",
+            "disease": "colorectal cancer [DOID:9256]",
+            "kbStatementId": "#999:999",
+            "kbVariant": "APC:p.T1556fs",
+            "variant": "a12458ff7c8eb865656fd61f670cd0d5",
+            "variantType": "mut",
+            "kbVariantId": "#158:1343",
+            "matchedCancer": true,
+            "reference": "pmid:TEST",
+            "relevance": "resistance",
+            "externalSource": "IPRKB",
+            "reviewStatus": "pending",
+            "kbData": {}
+        },
+        {
+            "approvedTherapy": false,
+            "category": "therapeutic",
+            "context": "test inhibitor",
+            "disease": "colorectal cancer [DOID:9256]",
+            "kbStatementId": "#999:999",
+            "kbVariant": "ZFP36L2:p.Q401del",
+            "variant": "TEST1",
+            "variantType": "mut",
+            "kbVariantId": "#999:9999",
+            "matchedCancer": true,
+            "reference": "pmid:TEST",
+            "relevance": "resistance",
+            "externalSource": "IPRKB",
+            "reviewStatus": "pending",
+            "kbData": {}
+        },
+        {
+            "approvedTherapy": false,
+            "category": "biological",
+            "context": "APC",
+            "kbStatementId": "#155:10801",
+            "kbVariant": "APC frameshift mutation",
+            "variant": "a12458ff7c8eb865656fd61f670cd0d5",
+            "variantType": "mut",
+            "kbVariantId": "#161:769",
```

Comment:
Further down (line 391 and others), kbStatementId is assigned a long description; I suppose it's meant to be an RID now?